tests/fsstress/testdata/fsstress_test.txt

```diff
@@ -0,0 +1,31 @@
+{{$test_opts := "-test.v -name fsstress-app"}}
+
+# Starting of reboot detector with a 1 reboot limit
+! test eden.reboot.test -test.v -timewait 45m -reboot=0 -count=1 &
```

Comment:
Please adjust timewait here to be close to the whole test time.

Reply:
Done

Reply:
Thank you